neuralop/models/uno.py

```diff
@@ -144,7 +143,8 @@ def __init__(self,
         
 
         if domain_padding is not None and domain_padding > 0:
-            self.domain_padding = DomainPadding(domain_padding=domain_padding, padding_mode=domain_padding_mode, output_scale_factor = output_scale_factor)
+            self.domain_padding = DomainPadding(domain_padding=domain_padding, padding_mode=domain_padding_mode\
+            , output_scale_factor = [i['res_scaling'] for i in layer_configs])
```

Comment:
Why the list comprehension here? We should accept (and use) a single float for all the spatial dimensions. Otherwise we should probably just have a list directly passed in config.

Reply:
as each layer of UNO has a different (possibly fractional) scaling factor, using a fixed output scale factor creates issues due to rounding for float value. So the actual output resolution may vary depending on the input resolution. 
So, for UNO, the Padding class is provided with the domain scaling factor along each dimension for all the layers.
Added support for output_scale_factor as a scaler and None type.

Reply:
But each layer should have a list, this looks like each layer has an int? 
i.e. layer_config['res_scaling'] should be a list directly?